coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -115,3 +141,33 @@ def get_ignore_comment(self, origin, language):
                         language=language))
                 ignore_comment = None
         return ignore_comment
+
+    def read_ignore_comments(self, original_line, start_comment,
```

Comment:
https://github.com/coala/coala/pull/5069#discussion_r185209788 not done yet.

As discussed on private chat, maybe create a new module for new ignore-related code to live in.